js/google_shopping_functions.js

```diff
@@ -5,14 +5,91 @@
  * input: accepts full item data
  * output: returns the length of the items array
  */
+
 function getItemsCount(itemData) {
   return itemData.items.length;
 }
 
 /*
  * Define and use your functions here
  */
+function getItems(objectData) {
+  if (objectData.items) {
+    //Check if items object exist
+    return objectData.items;
+  }
+}
+function getItemsByBrand(items, brand) {
+    let itemArr = [];
+    for (var i =0; i<items.length; i++){
+        if (items[i].product.brand === brand) {
+          itemArr.push(items[i]);
+        }
+    }
+    return itemArr;
+}
+function getItemsByAuthor(items, author) {
+    let itemArr = [];
+    for (var i = 0; i < items.length; i++) {
+      //if (items[i].product.author.name === author) {
+        if (items[i].product.author.name.includes("eBay")) {
+        itemArr.push(items[i]);
+      }
+    }
+    return itemArr;
+}
+function getAvailableProducts(items) {
+  let itemArr = [];
+  for (var i = 0; i < items.length; i++) {
+    if (items[i].product.inventories[0].availability === "inStock") {
+      itemArr.push(items[i]);
+    }
+  }
+  return itemArr;
+}
+
+//try {
+  //Check if file exists
+  var data = require("../products.json");
```

Comment:
usually we put this line at the top of the file

Reply:
Got it. Thanks!